### DISCUSSION TITLE
Demo three roles

### DISCUSSION THREAD

**Discussion by H-e-n-r-i** [COS301-SE-2022/Exploring-Self-Sovereign-Identity#126] [General]
I'm doing everything frontend.

**Comment by MxolisiMkanzi** [comment]
I am updating/enhancing the API (Centralized), adding and fixing the API to be 100% functional and meet all the requirements

**Comment by H-e-n-r-i** [comment]
@JohanCSmit @RebeccaPillay @10416260 please make sure to fill in your roles.

**Comment by JohanCSmit** [comment]
Worked out
